python/cuml/benchmark/bench_helper_funcs.py

```diff
@@ -18,6 +18,8 @@
 from cuml.utils import input_utils
 import numpy as np
 import pandas as pd
+import pickle as pickle
+import sklearn.ensemble as ensemble
```

Comment:
Could you either keep the fully scoped name `sklearn.ensemble` or import as `skEnsemble`? Since cuML also has an ensemble module, it's easy to get confused.

Reply:
Done.